src/dependency-manager/src/service-config/v1.ts

```diff
@@ -177,12 +220,13 @@ export class ServiceConfigV1 extends ServiceConfig {
     return this.dockerfile;
   }
 
-  getDependencies(): { [s: string]: string } {
-    return this.dependencies || {};
+  getDependencies() {
+    return this.dependencies;
   }
 
   addDependency(name: string, tag: string) {
-    this.dependencies[name] = tag;
+    this.dependencies[name] = new ServiceConfigV1();
```

Comment:
This isn't really used, but still should add:

`this.dependencies[name].name = name;`